feature-libs/product-configurator/rulebased/root/interceptor/cpq-configurator-rest.interceptor.ts

```diff
@@ -22,6 +21,7 @@ export const CPQ_CONFIGURATOR_VIRTUAL_ENDPOINT =
 export class CpqConfiguratorRestInterceptor implements HttpInterceptor {
   protected readonly HEADER_ATTR_CPQ_SESSION_ID = 'x-cpq-session-id';
   protected readonly HEADER_ATTR_CPQ_NO_COOKIES = 'x-cpq-disable-cookies';
+  cpqSessionId: string | null;
```

Comment:
@Uli-Tiger TBH I'm in favour of keeping the interceptor stateless and storing any data (like `cpqSessionId`) in the `CpqAccessStorageService`. What do you think?

Reply:
done